app/containers/MessageBox/forceJpgExtension.test.ts

```diff
@@ -0,0 +1,121 @@
+import { forceJpgExtension } from './forceJpgExtension';
+
+interface Image {
```

Comment:
No need to create the interface, just import.
`import { Image } from 'react-native-image-crop-picker';`